src/tags/breadcrumbs/gb-breadcrumbs.ts

```diff
@@ -5,11 +5,19 @@ import { Events, Results } from 'groupby-api';
 export interface BreadcrumbsConfig {
   hideQuery?: boolean;
   hideRefinements?: boolean;
+  labels?: boolean;
+  resultsLabel?: string;
+  noResultsLabel?: string;
+  correctedResultsLabel?: string;
 }
 
 export const DEFAULT_CONFIG: BreadcrumbsConfig = {
   hideQuery: false,
-  hideRefinements: false
+  hideRefinements: false,
+  labels: true,
+  resultsLabel: 'Results for: ',
```

Comment:
I feel like the space should be something we add explicitly instead of requiring that they do it?
Maybe not?